src/main/java/Board.java

```diff
@@ -1,5 +1,53 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class Board {
-    public String getBoard() {
-        return "Hello world.";
+    public final List grid = new ArrayList<>(Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8, 9));
+
+    public static final int[][] allWinningCombinations = {{0, 1, 2}, {3, 4, 5}, {6, 7, 8}, {0, 3, 6}, {1, 4, 7}, {2, 5, 8}, {2, 4, 6}, {0, 4, 8}};
+
+    public Boolean isEmpty() {
+        return !grid.contains("X") && !grid.contains("O");
     }
-}
+
+    public Object markBoard(int position, String mark) {
+        return grid.set(position - 1, mark);
+    }
+
+    public Object getMarkAtPosition(int position) {
```

Comment:
It would be good to try avoid using `Object` as a type as much as possible.

In this case, your function signature says that it will return 'something'  when you call `getMarkAtPosition`; I think you could be more specific about what this function returns.

Can you think of reasons why having a more specific return type would be better?

Reply:
✅ now expect type `String` PR #6